.github/workflows/helm-chart-dependency-check.yml

```diff
@@ -7,28 +7,59 @@ jobs:
     env:
       MAGMA_ROOT: "${{ github.workspace }}"
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        charts:
-          - ["orc8r", "$MAGMA_ROOT/orc8r/cloud/helm/orc8r/"]
-          - ["cwf-orc8r", "$MAGMA_ROOT/cwf/cloud/helm/cwf-orc8r/"]
-          - ["lte-orc8r", "$MAGMA_ROOT/lte/cloud/helm/lte-orc8r/"]
-          - ["feg-orc8r", "$MAGMA_ROOT/feg/cloud/helm/feg-orc8r/"]
-          - ["fbinternal-orc8r", "$MAGMA_ROOT/fbinternal/cloud/helm/fbinternal-orc8r/"]
-          - ["wifi-orc8r", "$MAGMA_ROOT/wifi/cloud/helm/wifi-orc8r/"]
     name: Check dependency of helm chart ${{ matrix.charts[0] }}
     steps:
       - uses: actions/checkout@v2
-      - name: Get chart.lock digest
+      - name: Check Orc8r
         run: |
-          echo "DIGEST=$(cat ${{ matrix.charts[1] }}Chart.lock | grep digest | cut -d ":" -f 2-3 | xargs)" >> $GITHUB_ENV
-      - name: Run dependency check against ${{ matrix.charts[0] }}
+          echo "DIGEST=$(cat $MAGMA_ROOT/orc8r/cloud/helm/orc8r/Chart.lock | grep digest | cut -d ":" -f 2-3 | xargs)" >> $GITHUB_ENV
+          helm dependency update "$MAGMA_ROOT/orc8r/cloud/helm/orc8r/"
+          echo "NEW_DIGEST=$(cat $MAGMA_ROOT/orc8r/cloud/helm/orc8r/Chart.lock | grep digest | cut -d ":" -f 2-3 | xargs)" >> $GITHUB_ENV
+          if [ "$NEW_DIGEST" != "$NEW_DIGEST" ]; then
+            exit 1
+          fi
+      - name: Check cwf-orc8r
+        if: always()
```

Comment:
`if: always()` seems like we can just remove that line (?)

Reply:
No we can't because we want all the checks to run even if one fails before.

Reply:
Ah I see. Good to know, I didn't realize that was the default, thanks for explaining 🙂